src/cargo/core/features.rs

```diff
@@ -1297,10 +1297,10 @@ impl CliUnstable {
             "rustdoc-map" => self.rustdoc_map = parse_empty(k, v)?,
             "rustdoc-scrape-examples" => self.rustdoc_scrape_examples = parse_empty(k, v)?,
             "sbom" => self.sbom = parse_empty(k, v)?,
-            "script" => self.script = parse_empty(k, v)?,
             "separate-nightlies" => self.separate_nightlies = parse_empty(k, v)?,
             "checksum-freshness" => self.checksum_freshness = parse_empty(k, v)?,
             "skip-rustdoc-fingerprint" => self.skip_rustdoc_fingerprint = parse_empty(k, v)?,
+            "script" => self.script = parse_empty(k, v)?,
             "target-applies-to-host" => self.target_applies_to_host = parse_empty(k, v)?,
```

Comment:
Why was this moved?  I believe we generally try to be sorted here

Reply:
It was inadvertently moved in the 1st commit when merging. The 2nd commit fixes it.

If you prefer, I can squash the two commits together. I was hoping it would be easier to review only the changes since last time, so I added a commit rather than rewriting the 1st one.

Reply:
I've done enough of a pass; let's squash